runtime/compute/cker/include/cker/operation/Einsum.h

```diff
@@ -150,11 +150,11 @@ std::vector<std::string> strSplit(const std::string &text, const std::string del
     pos = text.find(delimiter, start);
     if (pos == std::string::npos)
     {
-      result.push_back(text.substr(start, text.size() - start));
+      result.push_back(std::string(text.substr(start, text.size() - start)));
       break;
     }
 
-    result.push_back(text.substr(start, pos - start));
+    result.push_back(std::string(text.substr(start, pos - start)));
```

Comment:
Using `std::string_view` internally can reduce unnecessary copying during the splitting process, which might improve performance in some scenarios. However, since each substring is eventually converted to an `std::string` for the returned vector, the benefit might be less pronounced if most of the work is spent on that conversion. In many cases, the performance difference compared to the original code will be minor, but for large input strings or in performance-critical contexts, avoiding extra copies could provide some speed improvements.